### ISSUE TITLE
doScroll ContentLoaded doesn't wait in iframe

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#87]
```
What steps will reproduce the problem?
1. Append iframe to document.body
2. Load src containing ie7.js reference
3. the "initialisation" errors as: 'body' is null or not an object

What is the expected output? What do you see instead?
This function must wait until content has loaded before executing (for png
fix and access to the 'body' object). Here, the 'doScroll' method is not
throwing an error, resulting in the premature execution of the following
statements.

What version of the product are you using? On what operating system?
"IE7 version 2.0 (beta2)" IE7.js (line 2186) & IE8.js (line 2581)
Windows XP SP2 - IE 6 (Virtual PC)
Windows XP SP2 - IE 7

Please provide any additional information below.
Alternative to IEContentLoaded method, catching the "complete" readyState
executes after the 'body' is loaded (pro: executed consistently before the
onload event; con: does not wait long enough for pngs to load [to avoid the
IE6 png-filter-lockup])

```

Original issue reported on code.google.com by `dark.acc...@gmail.com` on 6 Jun 2008 at 1:03

**Comment by GoogleCodeExporter** [comment]
```
Can you provide a test page please?
```

Original comment by `dean.edw...@gmail.com` on 11 Feb 2010 at 1:13

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in version 2.1.
```

Original comment by `dean.edw...@gmail.com` on 14 Feb 2010 at 5:17
- Changed state: **Fixed**
